Workbooks/Usage/Usage Calendar/Usage Calendar.workbook

```diff
@@ -143,26 +124,54 @@
                 "palette": "green"
               }
             }
+          ],
+          "labelSettings": [
+            {
+              "columnId": "0.0",
+              "label": "Sun"
+            },
+            {
+              "columnId": "1.0",
+              "label": "Mon"
+            },
+            {
+              "columnId": "2.0",
+              "label": "Tue"
+            },
+            {
+              "columnId": "3.0",
+              "label": "Wed"
+            },
+            {
+              "columnId": "4.0",
+              "label": "Thu"
+            },
+            {
+              "columnId": "5.0",
+              "label": "Fri"
+            },
+            {
+              "columnId": "6.0",
+              "label": "Sat"
+            }
           ]
         }
       },
-      "conditionalVisibility": null,
-      "customWidth": "65"
+      "customWidth": "55",
+      "name": "Usage table"
     },
     {
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let start = startofweek({TimeRange:start});\nrange i from 0d to 366d step 1d\n| extend timestamp = startofday(now() - i)\n| make-series Max = max(1) default=0 on timestamp in range(start, endofweek({TimeRange:end}), 1d)\n| mvexpand timestamp to typeof(datetime), Max\n| extend WeekDay = dayofweek(timestamp), Week = toint((timestamp - start) / 7d), Day = dayofmonth(timestamp)\n| where '{ExcludeWeekends}' == 'No' or (WeekDay != 0d and WeekDay != 6d)\n| extend WeekDay = case(WeekDay == 0d, 'Sun', WeekDay == 1d, 'Mon', WeekDay == 2d, 'Tue', WeekDay == 3d, 'Wed', WeekDay == 4d, 'Thu', WeekDay == 5d, 'Fri', 'Sat')\n| order by timestamp asc\n| project-away timestamp, Max\n| evaluate pivot(WeekDay, sum(Day))\n| project-away Week ",
-        "showQuery": false,
+        "query": "let start = startofweek({TimeRange:start});\nrange i from 0d to 366d step 1d\n| extend timestamp = startofday(now() - i)\n| make-series Max = max(1) default=0 on timestamp in range(start, endofweek({TimeRange:end}), 1d)\n| mvexpand timestamp to typeof(datetime), Max\n| extend WeekDay = dayofweek(timestamp) / 1d, Week = toint((timestamp - start) / 7d), Day = dayofmonth(timestamp)\n| where '{ExcludeWeekends}' == 'No' or (WeekDay != 0d and WeekDay != 6d)\n| order by WeekDay asc, Week asc\n| serialize\n| project-away timestamp, Max\n| evaluate pivot(WeekDay, sum(Day))\n| project-away Week ",
         "size": 0,
-        "aggregation": 0,
-        "showAnnotations": false,
-        "showAnalytics": false,
+        "queryType": 0,
         "resourceType": "microsoft.insights/components"
       },
-      "conditionalVisibility": null,
-      "customWidth": "35"
+      "customWidth": "45",
+      "name": "Usage calendar"
```

Comment:
Looks like we need a `gridSettings` object to have persist labels (maybe this is a workbook bug?). but for now can you check the checkbox for 'Custom formatting` and reapply the labels? otherwise the second calendar looks like this

![image](https://user-images.githubusercontent.com/7664931/91897124-a9e23580-ec4e-11ea-8b43-439809f6c2f5.png)

Reply:
Addressed code review comments.

![image](https://user-images.githubusercontent.com/9730127/98410345-5a8efb80-2029-11eb-9507-126795f679c7.png)

![image](https://user-images.githubusercontent.com/9730127/98410397-7397ac80-2029-11eb-9a37-49b11a40f625.png)